From a604625e96405fb6b753c3e8a720b13e3d34d471 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Tue, 28 Nov 2023 14:35:54 +0100
Subject: [PATCH 1/6] powerpc/boot/wrapper: Add "-z notext" flag for ppc64le

This is a follow up to commit 4c078c86b4a4 (powerpc/boot/wrapper: Add
"-z notext" flag to disable diagnostic)

Ref: https://gitlab.alpinelinux.org/alpine/aports/-/issues/14560
Signed-off-by: Achill Gilgenast <fossdd@pwned.life>
---
 arch/powerpc/boot/wrapper | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/powerpc/boot/wrapper b/arch/powerpc/boot/wrapper
index 3d8dc822282a..9d45a9a74b1c 100755
--- a/arch/powerpc/boot/wrapper
+++ b/arch/powerpc/boot/wrapper
@@ -266,6 +266,7 @@ pseries)
     if [ "$format" != "elf32ppc" ]; then
 	link_address=
 	pie=-pie
+	notext='-z notext'
     fi
     make_space=n
     ;;
-- 
2.50.0

